### ISSUE TITLE
Question regarding dataset tuning

### ISSUE THREAD

**Issue by bhaswa** [SeanNaren/deepspeech.pytorch#542]
Hi,
In the supplied pre trained models, you have mentioned that the models are heavily tuned to the datasets they are trained on. May I know what are the tuning you talking about? If I want to train a model on my dataset, what kind of tuning do I need to follow?

**Comment by SeanNaren** [comment]
Maybe I should change the wording in the docs, when I meant 'tuned' I mean't that they've been trained on just the corresponding training set. In many cases this means you'll get pretty reasonable results on the validation test set for the corresponding dataset, but don't expect the model to do well on any files that are outside this dataset.

What I mean is, the model is trained on librispeech, it's going to do alright on librispeech val/test sets. But if I come along with my own audio, it's not going to do well! To do this, you'll need to scale your dataset sizes to something that isn't covered in this repo (in the DeepSpeech2 paper, they required around 6 to 11k hours of supervised data).

**Comment by bhaswa** [comment]
@SeanNaren  Thanks for the explanation!!
